app/test_split_integer.py

```diff
@@ -2,20 +2,30 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value, number_of_parts = 17, 4
+    parts = split_integer(value, number_of_parts)
+    assert sum(parts) == value
```

Comment:
```suggestion
    assert sum(split_integer(17, 4)) == 17
```
In this case, it's not necessary to create variables.